### PR TITLE
add the coif

### LINKED ISSUE #2075: cm species are not reading their species clothing sprites; see reptilian coif or any mask

### PR DESCRIPTION
**Pull request by DangerRevolution**
## About the PR
title. this needs some extra race-based sprites to be made

- fixes #2075
- needed for #1971
#1893
#1836
#1721
#1600
#1546
#1387

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame

image
image
image
image
image
image
image
image

**Changelog**

:cl: DangerRevolution
- add: Added the Coif

### CODE REVIEW

**Comment by DangerRevolution** [comment]
gonna leave this as a draft because i don't really feel like spriting wider masks for Lizards & Vox right now.

**Comment by DangerRevolution** [comment]
okay, well, i've done the Reptile sprite but I *cannot* for the *life* of me get this thing to show up for Reptiles. Gonna test it more soon.

File: Resources/Textures/_CM14/Objects/Clothing/Mask/coif.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from CM-SS13 at https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/mask.dmi",
```
Comment (QuietlyWhisper) [inline Resources/Textures/_CM14/Objects/Clothing/Mask/coif.rsi/meta.json:4]: You just need to add onto this your two new sprites and that you modified them (they're considered your modified sprites)

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
Just one last thing before merge

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**